packages/frontend/src/components/ProjectRoute.tsx

```diff
@@ -39,7 +39,7 @@ const ProjectRoute: FC<ComponentProps<typeof Route>> = ({
                     !projectsRequest.data ||
                     projectsRequest.data?.length <= 0
                 ) {
-                    return <Redirect to={`/no-access`} />;
+                    return <Redirect to={`/createProject`} />;
```

Comment:
I think we should show the user the no access page so they can then ask for access to the admin. 
If the user would be confused if is trying to open a link someone sent him and is being redirected to the create page. 
iirc @PriPatel initially suggested the no access page to have a button to create a project. Don't know if is still relevant. I don't think this would be a blocker and would be fine if we don't add the button. [slack thread] (https://lightdash.slack.com/archives/C02NPFHF4F6/p1655282704502969?thread_ts=1655280035.501019&cid=C02NPFHF4F6)

Reply:
Yeah, I think you are right, I think adding the button should fix the issue while keeping the option to create project accessible. On the ticket it said `Change error page to have CTA to create a project` , so what should it be ? @PriPatel @owlas